src/style.css

```diff
@@ -1,3 +1,81 @@
+html {
+  color: #00acdc;
+}
+
 body {
   margin: 0;
+  box-sizing: border-box;
+}
+
+.header__container {
+  padding: 0 50px;
+  margin: 0 auto;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  background-color: white;
+  flex-wrap: wrap;
+  gap: 60px;
+  max-width: 1200px;
+}
+
+.logo__img {
+  width: 100%;
+  height: 40px;
+}
+
+.nav__list {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 20px;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 60px;
+  text-transform: uppercase;
+  list-style: none;
+  height: 60px;
+}
+
+.nav__link {
+  border-bottom: 4px solid transparent;
+  text-decoration: none;
+  transition:
+    color 0.5s,
+    border-color 0.5s;
+  color: black;
+}
+
+.nav__link:hover {
+  color: #00acdc;
+  border-bottom: 4px solid;
+}
+
+a.is-active {
+  position: relative;
+  color: #00acdc;
+}
+
+a.is-active:hover {
+  border-bottom: 4px solid transparent;
+}
+
+.is-active::after {
+  content: '';
+  display: block;
+  width: 100%;
+  height: 4px;
+  background-color: #00acdc;
+  position: absolute;
+  border-radius: 8px;
+  bottom: 0;
+  left: 0;
+}
+
+.nav__link:nth-child(4) {
+  color: #00acdc;
+}
```

Comment:
The selector `.nav__link:nth-child(4)` is used to style the fourth navigation link. This approach is fragile because it relies on the order of elements in the HTML. Consider using a more specific class or data attribute to target this element.